src/main/kotlin/org/radarbase/push/integration/common/user/Users.kt

```diff
@@ -0,0 +1,19 @@
+package org.radarbase.push.integration.common.user
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.radarbase.push.integration.garmin.user.GarminUser
+import java.util.*
+
+
+class Users @JsonCreator constructor(@JsonProperty("users") users: List<GarminUser?>?) {
```

Comment:
this whole class can be replaced with
```kotlin
data class Users(val users: List<GarminUser?>)
```

I'm not sure why the GarminUser can be null, it would be even better to have
```kotlin
data class Users(val users: List<GarminUser>)
```